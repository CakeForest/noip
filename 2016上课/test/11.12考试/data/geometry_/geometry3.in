10
16 76 52 16 44 68 75 20 15 12 
88 78 20 26 88 30 38 81 23 42 
10
81 92
68 70
23 53
92 64
68 8
12 60
15 0
38 32
0 28
76 16
