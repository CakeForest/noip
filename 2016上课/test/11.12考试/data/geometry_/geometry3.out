10
10
7
10
8
7
2
7
3
10
